build/version.gni

```diff
@@ -1,27 +1,11 @@
-# Copyright 2015 The Chromium Authors. All rights reserved.
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
```

Comment:
You're supposed to add copyright notices, not replace existing ones -- even if you change it, the code came from Chromium.